blobfuse/src/AttrCacheBfsClient.cpp

```diff
@@ -528,4 +528,18 @@ void AttrCacheBfsClient::GetExtraProperties(const std::string pathName, BfsFileP
 int AttrCacheBfsClient::RefreshSASToken(std::string sas)
 {
     return blob_client->RefreshSASToken(sas);
+}
+
+
+void AttrCacheBfsClient::InvalidateFile(const std::string blob)
+{
```

Comment:
Either add InvalidateDir or remove this from this PR, since the request is invalidate both a file and a directory

Reply:
Added and invalidating attr-cache as per the task item.